src/sched.rs

```diff
@@ -35,6 +35,39 @@ bitflags!{
     }
 }
 
+#[derive(Clone, Copy, Debug)]
+pub struct CloneFlagsArg {
+    flags: CloneFlags,
+    signal: Option<c_int>,
+}
+
+impl CloneFlagsArg {
+    fn bits(self) -> c_int {
+        self.flags.bits() | self.signal.unwrap_or(0)
```

Comment:
Good point. It may not be necessary if we use an enum for signals though.... will toy with it over the next few days.